pyxem/signals/reduced_intensity_profile.py

```diff
@@ -0,0 +1,168 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017-2019 The pyXem developers
+#
+# This file is part of pyXem.
+#
+# pyXem is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# pyXem is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with pyXem.  If not, see <http://www.gnu.org/licenses/>.
+"""Signal class for Reduced Intensity profiles as a fucntion of scattering
+vector.
+
+"""
+
+from hyperspy.signals import Signal1D
+# ??from hyperspy.component import Polynomial
+import numpy as np
+from scipy import special
+from scipy.signal import savgol_filter
+
+from pyxem.components.reduced_intensity_correction_component import ReducedIntensityCorrectionComponent
+
+
+class ReducedIntensityProfile(Signal1D):
+    _signal_type = "reduced_intensity_profile"
+
+    def __init__(self, *args, **kwargs):
+        Signal1D.__init__(self, *args, **kwargs)
+
+    def damp_exponential(self, b):
```

Comment:
I think it would be better to make one "damp" method in this class with a kwarg to choose which one is used. The code syntax should be a bit like what is used for the peak finders.

Reply:
Let's discuss on Monday.

Reply:
So as discussed offline - we'll merge this PR with these as separate functions and then have a new PR in which we create a structure more like find_peaks with a little GUI too.